src/test/java/com/artipie/repo/ConfigFileTest.java

```diff
@@ -38,26 +40,65 @@
  * Test cases for {@link ConfigFile}.
  * @since 0.14
  */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
 final class ConfigFileTest {
 
     /**
      * Filename.
      */
     private static final String NAME = "my-file";
 
+    /**
+     * Content.
+     */
+    private static final byte[] CONTENT = "content from config file".getBytes();
+
     @ParameterizedTest
     @CsvSource({".yaml", ".yml", "''"})
-    void existInStorageReturnsWhenYamlExist(final String extension) {
+    void existInStorageReturnsTrueWhenYamlExist(final String extension) {
         final Storage storage = new InMemoryStorage();
         this.saveByKey(storage, ".yaml");
         MatcherAssert.assertThat(
             new ConfigFile(new Key.From(ConfigFileTest.NAME + extension))
                 .existsIn(storage)
-                .toCompletableFuture().join(),
+                .join(),
             new IsEqual<>(true)
         );
     }
 
+    @ParameterizedTest
+    @CsvSource({".yaml", ".yml", "''"})
```

Comment:
@genryxy `@CsvSource` is not needed here, it's easier to use `@ValueSource` for single argument tests